### PR TITLE
Readd dropped application subclass example

### PR DESCRIPTION
**Pull request by Hofer-Julian**
It also renames the screenshot names and removes
the in-source descriptions in order to enforce the README
as single source of truth.

@YaLTeR @bilelmoussaoui

### CODE REVIEW

File: examples/video_player/README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 This example demonstrates how to use `gtk::Video` to play videos.
 
-![Screenshot](app.png)
\ No newline at end of file
+![Screenshot](screenshot.png)
```
Comment (bilelmoussaoui) [inline examples/video_player/README.md:5]: the file seems to have a double extension

Comment (bilelmoussaoui) [reply examples/video_player/README.md:5]: all of them actually

Comment (Hofer-Julian) [reply examples/video_player/README.md:5]: Ah, will fix it

Comment (Hofer-Julian) [reply examples/video_player/README.md:5]: Fixed

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**